### PR TITLE
make website content more visible

### PR DESCRIPTION
**Pull request by gregbunce**
this pr: 
- adds anchor links to the '[how to connect to the open sgid](https://gis.utah.gov/sgid/open-sgid/)' page, to more easily link to the different application examples (qgis, arcgis pro, etc) 
- adds links to the [open sgid connection page](https://gis.utah.gov/sgid/open-sgid/) to blog posts and [TOS page](https://gis.utah.gov/about/policy/open-sgid/)
- adds links to udots lrs info doc to the [lrs data page](https://gis.utah.gov/data/transportation/roads-system/#HighwayLinearReferencingSystemRoutes)
- adds links to the '[contribute to the sgid via agol' page](https://gis.utah.gov/gis-data-sharing/open-data/) to the data page and the [data sharing page](https://gis.utah.gov/gis-data-sharing/)
- makes the sgid index section on the [data page](https://gis.utah.gov/data/) pop, giving it a background color

### CODE REVIEW

**Comment by gregbunce** [comment]
my apologies in advance, that may have been a bit much for one pr.  let me know if you have any questions.  i'd be happy to walk anyone through the high-level details/reasoning.

**Comment by steveoh** [comment]
Hit ready for review if you are done drafting :)

**Comment by gregbunce** [comment]
Oops, thx!

File: _posts/2020-05-04-introducing-open-sgid.md [hunk 1]
```diff
@@ -20,18 +20,18 @@ It seems so long ago now, but it was less than a year ago when we first [hinted
...
+The timing could not be more perfect with the sudden increase in teleworking, which is not expected to wane any time soon, especially here in Utah with the implementation of the [Governor’s teleworking initiative](https://governor.utah.gov/2019/07/16/state-of-utah-introduces-teleworking-program-for-employees/). Not only does the Open SGID benefit Utahns, but it also signals the return to a truly publicly available, [direct database connection that can be utilized on a multitude of GIS platforms]({% link sgid/open-sgid/index.md %}). Furthermore, it initiates the sunsetting of the state network-only SGID10 database, although both databases (i.e., the Open SGID and the SGID10) will run in parallel for some time to allow users to transition and adapt to the Open SGID.
```
Comment (steveoh) [inline _posts/2020-05-04-introducing-open-sgid.md:23]: **praise**: way to sneak that link in there :)

File: data/transportation/roads-system/index.html [hunk 2]
```diff
@@ -141,6 +141,7 @@ <h3 id="HighwayLinearReferencingSystemRoutes">Highway Linear Referencing System
     <h5>Related Resource</h5>
     <ul class="dotless">
       <li><a href="http://data.udot.utah.gov">data.udot.utah.gov</a>
+        <li><a href="https://drive.google.com/file/d/1t03AWYHeqTjssTSKnzn60ygdIB-nqEAF/view">Read about the changes coming to the LRS</a>
```
Comment (steveoh) [inline data/transportation/roads-system/index.html:144]: **nitpick**: try not to leave extra spaces at the end of lines

```suggestion
        <li><a href="https://drive.google.com/file/d/1t03AWYHeqTjssTSKnzn60ygdIB-nqEAF/view">Read about the changes coming to the LRS</a>
```

File: data/index.html [hunk 3]
```diff
@@ -144,7 +144,9 @@ <h2>The Open SGID</h2>
         <p class="section-read-more">
             <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get connected</a>
         </p>
-        <h2>SGID Index</h2>
+    </div>
```
Comment (steveoh) [inline data/index.html:147]: **nitpick**: removing extra whitespace

```suggestion
    </div>
```

File: data/index.html [hunk 4]
```diff
@@ -154,6 +156,8 @@ <h2>SGID Index</h2>
         <p class="section-read-more">
             <a href="{% link data/sgid-index/index.html %}" class="btn">SGID Index</a>
         </p>
+    </div>
```
Comment (steveoh) [inline data/index.html:159]: **nitpick**: removing extra whitespace
```suggestion
    </div>
```

File: sgid/open-sgid/index.md [hunk 5]
```diff
@@ -25,7 +25,12 @@ To search the SGID, please visit the [SGID Index]({% link data/sgid-index/index.
 ### How to Connect
 {: .text-left }
 
-There are a variety of clients and applications that can connect to a PostgreSQL database. Here's how to connect using a few of the more common applications (ArcGIS Desktop, ArcGIS Pro, DBeaver, and QGIS).
+There are a variety of clients and applications that can connect to a PostgreSQL database. Here are a few of the more common applications and how to connect.
+
+- [ArcGIS Desktop](#arcgis-desktop)
+- [ArcGIS Pro](#arcgis-pro)
+- [DBeaver](#dbeaver)
+- [QGIS](#qgis)
```
Comment (steveoh) [inline sgid/open-sgid/index.md:34]: **praise**: this is a great addition

File: _posts/2020-05-04-introducing-open-sgid.md (hunk 1 above)
Comment (gregbunce) [reply _posts/2020-05-04-introducing-open-sgid.md:23]: ha!  thanks!

File: sgid/open-sgid/index.md (hunk 5 above)
Comment (gregbunce) [reply sgid/open-sgid/index.md:34]: thanks again!

**Pull request merged**